### PR TITLE
case 20334: keyboard keys to local entities and bugs fixes.

### PR DESCRIPTION
**Pull request by danteruiz**
Fixed the issue of keyboard keys being rotated weirdly in the login flow and also converted keyboard anchor and keys to be LocalEntities.

Ticket - https://highfidelity.manuscript.com/f/cases/20334/HMD-Login-Zone-tilted-keyboard

### CODE REVIEW

**Comment by danteruiz** [comment]
Test plan
- Please run the test rail plans for the Keyboard alone and the keyboard in HMD login zone
- confirm that they both work like before.

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/16371/HighFidelity-Beta-PR14685-d2bd453.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/16371/HighFidelity-Beta-PR14685-d2bd453.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/16371/HighFidelity-Beta-Interface-PR14685-d2bd453.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/16371/HighFidelity-Beta-PR14685-d2bd453.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/16371/HighFidelity-Beta-Interface-PR14685-d2bd453.exe

**Comment by JeffClinton** [comment]
Currently testing

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/16428/HighFidelity-Beta-PR14685-233fa9a.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/16428/HighFidelity-Beta-Interface-PR14685-233fa9a.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/16428/HighFidelity-Beta-PR14685-233fa9a.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/16428/HighFidelity-Beta-Interface-PR14685-233fa9a.exe

**Comment by JeffClinton** [comment]
Currently testing

File: interface/src/ui/Keyboard.cpp [hunk 1]
```diff
@@ -336,38 +343,45 @@ void Keyboard::updateTextDisplay() {
...
+        { "visible", raise }
+    };
 
-        auto backPlateOverlay = std::dynamic_pointer_cast<Cube3DOverlay>(overlays.getOverlay(_backPlate.overlayID));
+    overlays.editOverlay(_textDisplay.overlayID, textDisplayProperties);
 
-        if (backPlateOverlay) {
-            backPlateOverlay->setVisible(raise);
-        }
+    auto backPlateOverlay = std::dynamic_pointer_cast<Cube3DOverlay>(overlays.getOverlay(_backPlate.overlayID));
```
Comment (SamGondelman) [inline interface/src/ui/Keyboard.cpp]: Why are _backPlate and _textDisplay still overlays?

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/16696/HighFidelity-Beta-PR14685-b108076.apk

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/16696/HighFidelity-Beta-PR14685-b108076.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/16696/HighFidelity-Beta-Interface-PR14685-b108076.exe

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/16699/HighFidelity-Beta-PR14685-957b32a.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/16699/HighFidelity-Beta-PR14685-957b32a.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/16699/HighFidelity-Beta-Interface-PR14685-957b32a.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/16699/HighFidelity-Beta-PR14685-957b32a.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/16699/HighFidelity-Beta-Interface-PR14685-957b32a.exe

**Comment by JeffClinton** [comment]
Currently testing

**Comment by danteruiz** [comment]
Samuel is going merge this PR in his removal of overlays PR

**Pull request closed without merging**